bears/general/AnnotationBear.py

```diff
@@ -157,26 +228,27 @@ def get_range_end_position(file,
                                position,
                                func,
                                single_comment=False):
-        _range = end_position = None
+        selected_annotation = end_position = selected_end_annotation = None
         for annotation in annotations.keys():
             if text[position:].startswith(annotation):
+                selected_annotation = annotation
                 if not single_comment:
-                    ret_val = func(file,
-                                   filename,
-                                   text,
-                                   annotation,
-                                   annotations[annotation],
-                                   position)
+                    selected_end_annotation = annotations[selected_annotation]
```

Comment:
why not just directly use `annotations[annotation]` instead of having a separate variable?